apps/files/src/components/Files-App.vue

```diff
@@ -1,257 +1,339 @@
+                                            // }
+                                        ]
+                                    },
+                                    from: false
+                                },
+
+                                starred: false,
+
+                                mdate: file['fileInfo']['{DAV:}getlastmodified'],
+
+                                cdate: '',    //TODO: Retrieve data of creation of a file
+
+                                size: function () {
+                                    if (file.type === 'dir') {
+                                        return file['fileInfo']['{DAV:}quota-used-bytes'] / 100
+                                    } else {
+                                        return file['fileInfo']['{DAV:}getcontentlength'] / 100
+                                    }
+                                }(),
+
+                                extension: (file.type === 'dir') ? false : '',
+
+                                name: function () {
+                                    let pathList = file.name.split("/").filter(e => e !== "")
+                                    return pathList[pathList.length - 1];
+                                }(),
+
+                                path: file.name,
+
+                                id: file['fileInfo']['{DAV:}getetag']
+                            });
+                        });
+
+                        this.self = files.self;
+
+                        this.loading = false;
+
+                        this.resetSelect();
+                    }, delay)
+                }).catch(error => {``
+                    console.log(error);
+                    UIkit.notification({
+                        message: error.statusText,
+                        status: 'danger',
+                        pos: 'top-center'
+                    });
+                });
+            },
+            singleSelect(item) {
+                this.selected = [item];
+            },
+            resetSelect() {
+                this.selected = [];
+            },
+            multiSelect(item) {
+                if (this.isChecked(item))
+                    _.remove(this.selected, item);
+                else
+                    this.selected.push(item);
+            },
+            isChecked(item) {
+                return _.includes(this.selected, item);
+            }
+        },
+        watch: {
+            item() {
+                this.loadFolder()
+            }
+        },
+        computed: {
+            item() {
+                return this.$route.params.item;
+            },
+            typeOfFolder() {
+                return _.filter(this.files, ['extension', false])
+            },
+            typeOfFile(showHidden) {
+                showHidden = (typeof showHidden !== 'undefined') ? showHidden : false;
+                return _.filter(this.files, 'extension')
+            }
+        }
+    }
 </script>
 
 <style lang="less">
```

Comment:
I'd recommend against "larger" styling inside the vue files and rather have that move to a separate Less file, but we can do this in the future.

Reply:
They were there initially. I will start separating them.